src/main/java/titanicsend/osc/CrutchOSC.java

```
@@ -302,18 +326,19 @@ private void unregisterParameters() {
 
     private void unregisterParameter(LXListenableNormalizedParameter parameter) {
       if (parameter != null) {
-        parameter.removeListener(this);
+        try {
+          parameter.removeListener(this);
+        } catch (Exception ex) {
+          // Parameter was in remote controls twice? Look the other way...
+          // Don't mind the red text in the log.
```

if we're silently catching the exception, we won't log anything right? unless there something being logged inside the remove listener method. is this something that happens? genuinely curious why we're adding these but remove the other try/catches

---

Yeah this captures a funky known edge case.  In theory a user can edit the remote controls in the UI and add a parameter twice, but a listener can't register or unregister twice.  This could be made more graceful by tracking the registered parameters in a list.  Looks like I went for the fast hack solution, swallow the error when there's one possible cause and it's not a problem.
I'll make a note to improve this when I get to the TouchOSC side.